sample-apps/react-native/dogfood/src/screens/LiveStream/HostLiveStream.tsx

```diff
@@ -58,15 +133,72 @@ export const HostLiveStreamScreen = ({ route }: HostLiveStreamScreenProps) => {
 
   return (
     <StreamCall call={call}>
-      <SafeAreaView style={styles.container}>
-        <HostLivestream />
-      </SafeAreaView>
+      <BottomSheetModalProvider>
+        <Animated.View style={[styles.animatedContainer, animatedStyles]}>
+          <SafeAreaView edges={['top']} style={styles.container}>
+            <HostLivestream
+              HostLivestreamTopView={
+                !headerFooterHidden ? HostLivestreamTopView : null
+              }
+              // eslint-disable-next-line react/no-unstable-nested-components
+              LivestreamMediaControls={() => (
+                <LivestreamMediaControls
+                  handlePresentModalPress={handlePresentModalPress}
+                />
+              )}
```

Comment:
this is such a bad practice.. dont pass nested component like this

you should have created this component using useCallback

Reply:
Pushed 👍🏻